Lesson14/Lesson14-Busra/Busra.js

```diff
@@ -0,0 +1,185 @@
+const userResponse = prompt("What is your age?");
+const userResposeNumber = parseInt(userResponse);
+
+if (isNaN(userResposeNumber)) {
+  throw new Error("Please enter a valid number");
+} else {
+  console.log('userResponse',  Math.floor(userResposeNumber));
+}
+
+// 13. Declare a boolean variable isStudent = true. Log it.
+const isStudent = true;
+console.log(isStudent);
+
+// 14. Initialize counter = 0, then increment it by 1 using counter++ and log it.
+const counter = 0;
+counter++;
+console.log(counter);
+
+// 15. Initialize points = 10, add 5 to it using points += 5, then log points.
+const points = 10;
+points+=5;
+console.log(points);
+
+// 16. Declare name="Alice", age=30, city="Paris". Log "Alice (30) lives in Paris" using template literals.
+
+const nameExercise16 = "Alice"; //it gives an error because the "name" variable was previously defined with a different content,so I defined a different variable name
+const age = 30;
+const cityExercise16 = "Paris";
+console.log(nameExercise16 + " (" + age + ") " + "Lives in " + cityExercise16);
+
+// 17. Declare variables x=5, y=10, z=15. Log their total sum.
+const x = 5;
+const y = 10;
+const z = 15;
+const sum = x+y+z;
+console.log(sum);
+
+// 18. Declare dividend=10 and divisor=3. Log the quotient (divisionResult) and difference (differenceResult).
+const dividend = 10;
+const divisor = 3;
+const divisionResult = dividend / divisor;
+console.log("Bölüm sonucu:" + divisionResult);
+const differenceResult = dividend - divisor;
+console.log("Fark sonucu:" + differenceResult);
+
+// 19. Declare firstName and lastName. Create fullName by concatenating them with a space and log it.
+const firstName = "Busra";
+const lastName = "Demirkesen";
+console.log(firstName + " " + lastName);
+
+// 20. Declare firstFactor=7 and secondFactor=2. Log the product.
+
+const firstFactor = 7;
+const secondFactor = 2;
+
+console.log(firstFactor * secondFactor);
+
+// 21. Log the value of Math.PI.
+console.log(Math.PI);
+
+// 22. Declare counter=0. Increment it using three different methods (e.g., counter++, counter+=1, counter=counter+1) and log the result each time.
+
+const counterEx22 = 0;
+
+counterEx22 = counterEx22 + 1; // first method
+console.log(counter);
+
+counter +=1;  // second method
+console.log(counter);
+
+counter++; // third method
+console.log(counter);
+
+// 23. Declare initialTemperature=20. Increase it by 5 and log the result.
+const initialTemperature = 20;
+initialTemperature = initialTemperature + 5;
+console.log(initialTemperature);
+
+// 24. Declare numberEx9=6. Increment it using the prefix ++ operator and log both the variable and the incremented value.
+const numberEx9 = 6;
+console.log(numberEx9);
+console.log(++numberEx9);
+
+// 25. Declare numberEx10=8. Increment it using the postfix ++ operator and log both the original variable and the incremented value.
+const numberEx10 = 8;
+console.log(numberEx10);
+console.log(++numberEx10);
```

Comment:
You need to increment the value using postfix operator, so it will be: `numberEx10++`